src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -65,8 +65,10 @@
   ::transaction-completed
   (fn [{db :db} [_ {:keys [id response]}]]
     (let [{:keys [hash error]} response]
-      (when-not (and error (string? error) (not (string/blank? error)))
+      (if (and error (string? error) (not (string/blank? error)))
+        {:db (assoc-in db [:wallet/send-transaction :in-progress?] false)}
         {:db       (-> db
```

Comment:
```clojure
(-> db
    (update :wallet/send-transaction assoc 
            :in-progress? false
            :transaction-id nil)
    (assoc :wrong-password? false))
```

Reply:
but wait, `:in-progress?` is `false` in both branches, so we should update `db` before this `if`